prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'James Aguirre';
```

Comment:
You want to declare all of the variables in this file using the `const` keyword except for the `never` variable which should be declared using `let`